docs/developer_guide/pull-request.md

```diff
@@ -0,0 +1,34 @@
+# Creating a Pull Request
+
+Once you've made your changes to the MedModels codebase and are ready to share them, you'll submit a pull request (PR). Here's a guide to walk you through the process:
+
+**1. Push your Local Branch:**
+
+- Commit your changes locally using clear and concise commit messages following the Conventional Commits standard ([https://www.conventionalcommits.org/en/v1.0.0/](https://www.conventionalcommits.org/en/v1.0.0/)).
+- This standard encourages using a specific format for commit messages that categorizes your changes. Here's the breakdown:
+
+  - **Type:** Start your message with type like `feat`, `fix`, `refactor`, `docs`, etc. We use the following types in this project:
```

Comment:
Maybe it would sound better this way: _Start your message stating the PR's type_